refinery/tool_manager/models.py

```diff
@@ -806,6 +1062,18 @@ def update_galaxy_data(self, key, value):
         tool_launch_config[self.GALAXY_DATA][key] = value
         self.set_tool_launch_config(tool_launch_config)
 
+    def _update_galaxy_to_refinery_file_mapping_list(
+            self, galaxy_to_refinery_mapping_dict):
+        galaxy_to_refinery_mapping_list = (
+            self._get_galaxy_file_mapping_list()
+        )
+        galaxy_to_refinery_mapping_list.append(galaxy_to_refinery_mapping_dict)
+        self.update_galaxy_data(
+            self.GALAXY_TO_REFINERY_MAPPING_LIST,
+            galaxy_to_refinery_mapping_list
+        )
+
+    @handle_bioblend_exceptions
```

Comment:
not convinced that `handle_bioblend_exceptions` is necessary or implemented correctly (do we re raise these caugh exceptions)